File: Beecrowd/1009 Salário com Bônus/main.c
#include <stdio.h>
#include <string.h>

int main() {

    char nome[10];
    double salario, totalV, TOTAL;

    fflush(stdin);
    gets(nome);

    scanf("%lf %lf", &salario, &totalV);

    TOTAL = salario + totalV * 0.15;

    printf("TOTAL = R$ %.2lf\n", TOTAL);

    return 0;
}

File: Beecrowd/1009 Salário com Bônus/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        String nome;
        double salario, totalV, TOTAL;

        nome = sc.next();

        salario = sc.nextDouble();
        totalV = sc.nextDouble();

        TOTAL = salario + totalV * 0.15;

        System.out.printf("TOTAL = R$ %.2f%n", TOTAL);

        sc.close();
    }
}